ionic_user_interface/src/views/UserRoutes.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+  <ion-page>
+    <ion-content :fullscreen="true">
+      <ion-grid>
+        <ion-row class="ion-align-items-center ion-justify-content-center">
+          <ion-col class="ion-align-self-center" size-lg="6" size-md="8" size-xs="12">
+            <div class="page-title">
+              <b>Routes By User</b>
+            </div>
+            <ion-item class="rounded">
+              <ion-icon slot="start" :icon="personCircleOutline"></ion-icon>
+              <ion-text>{{ usernameText }}</ion-text>
+              <ion-icon
+                v-if="isLoggedIn && !isOwnself"
+                class="report-icon"
+                slot="end"
+                color="danger"
+                :icon="flagOutline"
+                @click="reportUserHandler"
+              ></ion-icon>
+            </ion-item>
+            <user-route-list />
```

Comment:
Need some way to update this, currently there's a bug when you click on someone else's userroutes page, then click `My Routes` - the route list doesn't update until you manually refresh the page.

Reply:
Possible solution is just to `watch(username, updateRoutes)` in UserRouteList.vue (haven't tested but should work).

Reply:
Ok tested it, you need to created a `computed(() => route.params.username)` for both UserRoutes and UserRouteList, then use that usernameRef.